server/src/main/java/org/elasticsearch/action/admin/cluster/stats/DenseVectorFieldStats.java

```diff
@@ -31,13 +37,9 @@ public final class DenseVectorFieldStats extends FieldStats {
         indexedVectorCount = 0;
         indexedVectorDimMin = UNSET;
         indexedVectorDimMax = UNSET;
-    }
-
-    DenseVectorFieldStats(StreamInput in) throws IOException {
```

Comment:
I noticed that this isn't actually used anywhere. I am going to double check that this is ok. But the underlying `fieldstats` serialization isn't via named writables & consequently the `DenseVectorFieldStats` is never actually written or read.

Reply:
+1 to remove